src/nvim/api/private/converter.c

```diff
@@ -57,7 +57,7 @@ typedef struct {
     const size_t len_ = (size_t)(len); \
     const blob_T *const blob_ = (blob); \
     kvi_push(edata->stack, STRING_OBJ(((String) { \
-      .data = len_ != 0 ? xmemdup(blob_->bv_ga.ga_data, len_) : NULL, \
+      .data = len_ != 0 ? xmemdupz(blob_->bv_ga.ga_data, len_) : xstrdup(""), \
```

Comment:
hmm, NULL is supposed to be allowed. Though perhaps there are too many places to fix?

Reply:
Well, there's definitely a few places :laughing:
Maybe viml string -> api string also allocs empty strings for that reason?

Reply:
Let's worry about it later..